apps/pie-docs/test/visual/home-page.spec.js

```diff
@@ -1,4 +1,4 @@
-import { test } from '../playwright/fixtures';
+import { test } from '@justeattakeaway/pie-webc-testing/src/playwright/playwright-fixtures';
```

Comment:
"pie-webc-testing" inside pie-docs 👀  is this expected?

Reply:
This is expected, yeah. I agree. The naming of this package could probably be better IMO.